5calls/app/src/main/java/org/a5calls/android/a5calls/util/CustomTabsUtil.java

```diff
@@ -0,0 +1,30 @@
+package org.a5calls.android.a5calls.util;
```

Comment:
(no action needed) if we make this package, maybe NotificationUtils should move into here too.